.github/packer/builder.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -euof
+
+install_deps() {
+    sudo yum install -y git docker make tar perl-Digest-SHA libicu
+}
+
+configure_docker() {
+    sudo systemctl enable docker --now
+    sudo usermod -aG docker ec2-user
+}
+
+build_goloop_images() {
+  git clone https://github.com/icon-project/goloop.git
+  cd goloop
+  make goloop-image
+  make goloop-icon-image
+  docker tag goloop iconloop/goloop-icon
+  rm -rf ../goloop
+}
+
+pull_archway_images() {
+  docker pull ghcr.io/archway-network/archwayd:v4.0.0
+}
+
+build_services() {
```

Comment:
It seems like we can utilize publicly available images for both goloop and archway. 

- iconloop/goloop-icon:latest
- archwaynetwork/archwayd:sha-8f53ac8

Reply:
We still need this image which is not available to public.

https://github.com/icon-project/IBC-Integration/blob/a455e5700becf7f3ab5cffd9187f65a489e03253/test/chains/icon/icon_node.go#L36C1-L37C1

Reply:
making a gochain image is a time-consuming process, at least on a local machine. Therefore, it might be more efficient to consider using publicly available images rather than rebuilding it each time. 

I believe @nightowl121 have publish gochain image on aws repo

Reply:
Following our discussion, it's clear that this image creation is a one-time setup, so it's acceptable to build the images initially.